### DISCUSSION TITLE
Move documentation to new repo to handle multiple sed versions

### DISCUSSION THREAD

**Discussion by zain-sohail** [OpenCOMPES/sed#452] [Ideas]
With v1.0 in plan, I think it's a good idea we keep the documentation for current 0.1.9 versions and so on as well. Likewise would be good to continue having docs for older verions. I suggest something like how numpy does it and store the doc builds in another repo https://github.com/numpy/doc

I already looked at the pydata sphinx theme and have a very test version working there (See https://github.com/OpenCOMPES/sed/tree/pydata-docs-theme). This theme looks/navigates way better and it has support for versioning (stable/latest/older as well)

Otherwise I find that storing the older doc versions would be tricky.
@rettigl

**Comment by rettigl** [comment]
Implemented
